src/main/java/uk/ac/ebi/eva/commons/mongodb/repositories/VariantRepositoryImpl.java

```diff
@@ -51,6 +52,8 @@
 
     private final int MARGIN = 5000;
 
+    private static final String GENE_DOC_PATH = VariantMongo.ANNOTATION_FIELD + "." + AnnotationIndexMongo.XREFS_FIELD;
```

Comment:
Could this constant be named more consistently with the rest of fields, i.e. `GENE_IDS_FIELD` or something like that?